iOS/RemoteCollectionController.swift

```diff
@@ -9,18 +9,36 @@ class RemoteCollectionController: UICollectionViewController, DateScrubberDelega
     override func viewDidLoad() {
         super.viewDidLoad()
 
+        for family: String in UIFont.familyNames()
+        {
+            print("\(family)")
+            for names: String in UIFont.fontNamesForFamilyName(family)
+            {
+                print("== \(names)")
+            }
+        }
+
         self.collectionView?.backgroundColor = UIColor.whiteColor()
+        
         self.collectionView?.registerClass(PhotoCell.self, forCellWithReuseIdentifier: PhotoCell.Identifier)
+        self.collectionView?.registerClass(SectionHeader.self, forSupplementaryViewOfKind: UICollectionElementKindSectionHeader, withReuseIdentifier: SectionHeader.Identifier)
+
+
         self.collectionView?.showsVerticalScrollIndicator = false
 
         var count = 0
-        for i in 0..<self.sections.count {
-            let photos = self.sections[i]
-            count += photos.count
+        for i in 0 ..< self.sections.count {
+            if let photos = self.sections[sectionTitleFor(i)] {
+                count += photos.count
+            }
         }
         self.numberOfItems = count
 
         self.dateScrubber.delegate = self
+        self.dateScrubber.scrubberImage = UIImage(named: "date-scrubber")
+        self.dateScrubber.sectionLabelImage = UIImage(named: "section-label")
+        self.dateScrubber.font = UIFont(name: "DINNextLTPro-Light", size: 18)
+        self.dateScrubber.textColor = UIColor(red: 69/255, green: 67/255, blue: 76/255, alpha: 0.8)
         self.view.addSubview(dateScrubber.view)
```

Comment:
All the stuff that you can customise from the client. If I document this in the readme and add a nice custom look when these aren't set it's oke right?